en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -212,6 +212,22 @@ Ethernet connectivity provides a fast, reliable, and flexible communication alte
 
 ::: info
 For more general information see: [PX4 Ethernet Setup](../advanced_config/ethernet_setup.md).
+Except we have used the below ```netplan``` config:
+
+```sh
```

Comment:
Just FYI (and you may have done this) I would add why you need this configuration. I presume it is because this is what comes "out of the box" and it is incompatible with PX4 out of the box. 

I won't make other comments until this is ready for review.

Reply:
The route is touched with the docs from Ethernet setup which disconnects internet access on pi. We either need to edit Ethernet Setup docs to this or just mention on this specific board. 
The best way to me is to correct ethernet setup doc.

Reply:
I added the reason for it under the note.

Reply:
Unfortunately, you might as well be speaking in another language. 

I think you are saying that the generic Ethernet docs cover other boards, and to make things correct for this board we need to do some things different. Right?

Reply:
> I think you are saying that the generic Ethernet docs assumes that the wired Ethernet connection is on `enp2s0` but for RPi it is `eth0` - is that right?

Correct!